lib/core/src/lib/comments/comment-list/comment-list.component.html

```diff
@@ -2,29 +2,22 @@
     <mat-list-item *ngFor="let comment of comments"
                   (click)="selectComment(comment)"
                   class="adf-comment-list-item"
-                  [class.adf-is-selected]="comment.isSelected"
-                  id="adf-comment-{{comment?.id}}">
-        <div [attr.id]="'comment-user-icon-' + comment.id" class="adf-comment-img-container">
-            <div
-                *ngIf="!isPictureDefined(comment.createdBy)"
-                class="adf-comment-user-icon">
-                {{getUserShortName(comment.createdBy)}}
+                  [class.adf-is-selected]="comment.isSelected">
+        <div class="adf-comment-img-container">
+            <div *ngIf="!isPictureDefined(comment.createdBy)"
+                 class="adf-comment-user-icon">
+                {{ getUserInitials(comment.createdBy) }}
             </div>
-            <div>
-                <img [alt]="comment.createdBy" *ngIf="isPictureDefined(comment.createdBy)"
-                      class="adf-people-img"
+            <div *ngIf="isPictureDefined(comment.createdBy)">
+                <img class="adf-people-img"
+                     [alt]="'COMMENTS.PROFILE_IMAGE' | translate"
                      [src]="getUserImage(comment.createdBy)" />
             </div>
         </div>
         <div class="adf-comment-contents">
-            <div matLine [attr.id]="'comment-user-' + comment.id" class="adf-comment-user-name">
-                {{comment.createdBy?.firstName}} {{comment.createdBy?.lastName}}
-            </div>
-            <div matLine [attr.id]="'comment-message-' + comment.id" class="adf-comment-message" [innerHTML]="comment.message">
-            </div>
-            <div matLine [attr.id]="'comment-time-' + comment.id" class="adf-comment-message-time">
-                {{ comment.created | adfTimeAgo: currentLocale }}
-            </div>
+            <div matLine class="adf-comment-user-name">{{ getUserFullName(comment.createdBy) }}</div>
```

Comment:
I see in that file we have many function calls (you added couple new like getUserFullName). Can we avoid that and refer to fields directly? We have jira for replacing all places (https://alfresco.atlassian.net/browse/ADF-5539) but if we can avoid adding new functions calls now then we will have less work. 

For example you can try to map comments list to list of different type objects and then refer to their fields instead of functions.

Reply:
I am try to keep the cleanup here, not rewriting the component or changing the scope of work. There's plenty of things here to be done, and if I start rewriting all the function calls, it will be definitely a big task that won't land any time soon

Reply:
moved some code to the Comment model